scripts/join_paired_ends.py

```diff
@@ -87,12 +87,12 @@
     make_option('-j', '--min_overlap', type='int',
                 help='Applies to both fastq-join and SeqPrep methods.' +
                       ' Minimum allowed overlap in base-pairs required to join pairs.' +
-                      ' If not set, progam defaults will be used.'
+                      ' If not set, progam defaults will be used. For example using fast-q-join (6 bp) will be used.'
```

Comment:
Can you change to:

*For example, for fastq-join (6 bp) will be used.*

(and make a similar change below).